clio-server/config/server.crt.ctmpl

```diff
@@ -0,0 +1 @@
+{{with $environment := env "ENVIRONMENT"}}{{with $commonSsl := vault (printf "secret/dsde/gotc/%s/common/server.crt" $environment)}}{{$commonSsl.Data.server_crt}}{{end}}{{end}}
```

Comment:
I'm a bit surprised there's no easier way to do this, but I know next to nothing about consul templates.